.changes/next-release/Test-148a692f-ec5a-4d58-b451-f04334e0779b.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Test",
+	"description": "change prompts for q feature dev e2e tests"
```

Comment:
Since this change isn't customer impacting I don't think we need a changelog item